plugins/tail.yml

```diff
@@ -0,0 +1,28 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: tail
+spec:
+  version: "v0.9.1"
+  platforms:
+  - uri: https://github.com/boz/kail/releases/download/v0.9.1/kail_0.9.1_darwin_amd64.tar.gz
+    sha256: 3824de171a73c773649401f2bdccbd138f0022333c69e39e4f194e8ebd08290b
+    bin: kail
+    files:
+    - from: "kail"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin]}
+  - uri: https://github.com/boz/kail/releases/download/v0.9.1/kail_0.9.1_linux_amd64.tar.gz
+    sha256: 6932b86f59c4b2cc73eb7aad889d3479566dfb4034d89a2d5f426722cdc5b780
+    bin: kail
+    files:
+    - from: "kail"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [linux]}
+  homepage: https://github.com/boz/kail
+  shortDescription: Kubernetes log viewer
+  description: Show logs from matched pods.  See homepage or `-h` for usage.
```

Comment:
The description will be shown when people want to know more before installing, e.g. `kubectl krew info tail` (btw, the homepage is a new addition and is not yet shown with the latest krew release).

You do yourself a favor, if you explain a bit more what are the typical use-cases for your tool and how to invoke it.

Reply:
If `-h` seems to be documenting the plugin well, we probably don’t need to explain usage string here.